python/thunder/imgprocessing/transformation.py

```diff
@@ -0,0 +1,58 @@
+""" Transformations produced by registration methods """
+
+
+class Transformation(object):
```

Comment:
What do you think about adding something like a `getParams()` and `setParams()` method that would be used to serialize the Transformation objects?